packages/theme/src/lib/light-theme.ts

```diff
@@ -598,6 +627,7 @@ export const lightTheme = {
       styleOverrides: {
         root: {
           '&.MuiInputBase-root': {
+            padding: '8px 12px',
```

Comment:
Have we identified all the effects of moving the padding from the input to the root? Looks like we're still using the `adornedEnd` etc items that alter the root vs swapping to `inputAdornedEnd` etc to ensure it all works correctly together.

I'm a bit nervous to move away from MUI patterns with a base class like this that could be difficult to change and account for all effects later down the line.

Reply:
I had to do this to get it to work with the Password Field example if I remember correctly. I hadn't checked it again after bringing in your changes. I'll see if it works without it

Reply:
I moved where the padding is applied. Changed it to use `MuiInputBase-input` and updated the padding for `MuiInputBase-adornedEnd` to match our padding of `12px`.